datahub-actions/tests/unit/cli/test_actions.py

```diff
@@ -96,6 +120,30 @@ def disabled_config_file() -> Generator[str, None, None]:
     os.unlink(config_path)
 
 
+@pytest.fixture
+def disabled_config_file_env_var() -> Generator[str, None, None]:
+    """Creates a temporary YAML config file with disabled pipeline."""
+    with tempfile.NamedTemporaryFile(mode="w", suffix=".yaml", delete=False) as f:
+        f.write(
+            """
+name: "disabled_pipeline"
+enabled: ${PIPELINE_ENABLED:-false}
+datahub:
+    server: "https://test.datahub.io"
+    token: "test-token"
+source:
+    type: "datahub-cloud"
+action:
+    type: "hello_world"
+"""
+        )
+        config_path = f.name
+
+    yield config_path
+    # Cleanup
+    os.unlink(config_path)
```

Comment:
the cleanup should happen automatically
```suggestion
    with tempfile.NamedTemporaryFile(mode="w", suffix=".yaml") as f:
        f.write(
            """
name: "disabled_pipeline"
enabled: ${PIPELINE_ENABLED:-false}
datahub:
    server: "https://test.datahub.io"
    token: "test-token"
source:
    type: "datahub-cloud"
action:
    type: "hello_world"
"""
        )
        yield f.name
```